distribution/pom.xml

```diff
@@ -124,6 +124,12 @@
             <version>${project.version}</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>farao-search-tree-rao</artifactId>
```

Comment:
What is the purpose of that module ?

Reply:
I think the farao-distribution is to generate a package of FARAO including itools (this is in description of this pom.xml). Seb will explain it better than me; but I think it will just deploy the .jar into the /share/ folder.